pages/1.11/deploying-services/containerizers/ucr/index.md

```diff
@@ -81,8 +81,21 @@ The [Universal Container Runtime (UCR)](http://mesos.apache.org/documentation/la
 
 **Important:** If you leave the `args` field empty, the default entry point will be the launch command for the container. If your container does not have a default entry point, you must specify a command in the `args` field. If you do not, your service will fail to deploy.
 
-# Limitations
-- The UCR does not support the following: runtime privileges, Docker options, private registries with container authentication.
+# Container Image Garbage Collection
+
+For long a long running cluster, container images may occupy disk spaces on the agent machines. To improve the operator's experience with UCR, container image GC is introduced, starting from Mesos 1.5.0 (please read [the Mesos docs](http://mesos.apache.org/documentation/latest/container-image/#garbage-collect-unused-container-images) for more details). The image GC is automatic by default in DC/OS while it can be triggered by the operator manually.
```

Comment:
Change:
"For long a long running cluster..."
to:
"For a long-running cluster..."